### ISSUE TITLE
H5P doesn't display at all after updating

### ISSUE THREAD

**Issue by Urpokarhu1** [h5p/h5p-course-presentation#98]
Hello,
 
after updating mod_hvp to 1.19.1 H5P-acitivities dissappear at random so when you open H5P-acitivity it's blank (browser doesn't matter happens on chrome and IE11) , sometimes they show and sometimes they don't. Only error is this from console

```
Uncaught TypeError: Cannot read property 'find' of undefined
    at S.setOverflowTabIndex (2209b9043998524046930798eb31a6e448cf220b.js:3775)
    at S.attachElement (2209b9043998524046930798eb31a6e448cf220b.js:3775)
    at i.t.appendElements (2209b9043998524046930798eb31a6e448cf220b.js:3775)
    at S.createSlides (2209b9043998524046930798eb31a6e448cf220b.js:3775)
    at S.attach (2209b9043998524046930798eb31a6e448cf220b.js:3775)
    at Object.H5P.newRunnable (h5p.js?ver=2019081600:944)
    at HTMLDivElement.<anonymous> (h5p.js?ver=2019081600:137)
    at Function.each (jquery.js?ver=2019081600:2)
    at init.each (jquery.js?ver=2019081600:2)
    at Object.H5P.init (h5p.js?ver=2019081600:93)
first.js:39165 Init add block button adder
DevTools failed to parse SourceMap: https://moodle/lib/requirejs.php/1566796628/core/popper.js.map
```

@thomasmars @icc

**Comment by Urpokarhu1** [comment]
Hello,

This problem sound like our: 

https://h5p.org/node/332953

**Comment by Urpokarhu1** [comment]
Steps to repeat with a new H5P course presentation:

1. Add a new h5p activity course presentation
2. Add 3 slides and text element on each page
3. Give H5P an title
4. Click Save and display
5. Open activity and go to the slide 3/3 (last slide)
6. Return to course mainpage from themes breadcrumb navigation
7. Open the activity and notice how it's empty

8. If you now edit H5P activity settings -> do not change everythin -> just click save and dipslay -> it now displays correctly

Tested with Chrome, IE11 and Firefox. @fnoks

**Comment by Urpokarhu1** [comment]
It seems if you disable enable_save_content_state this problem will not repeat

**Comment by fnoks** [comment]
I have reproduced, [fixed](https://github.com/h5p/h5p-course-presentation/commit/c3d86e094fc37e28b931aa5c445c33d03b8ed25b
) and released a new version of CP (1.21.2). Please upgrade, and let me know if it fixed the issues seen.



Thanks for reporting!

**Comment by anieminen** [comment]
Hi,

Upgrading Course presentation library seems to fix the problem. Thanks!
